src/main/java/uk/gov/ida/matchingserviceadapter/domain/HealthCheckResponseFromMatchingService.java

```diff
@@ -9,8 +9,8 @@
 
 public class HealthCheckResponseFromMatchingService extends IdaMatchingServiceResponse {
     
-    public HealthCheckResponseFromMatchingService(String entityId, String healthCheckRequestId, String msaVersion, boolean eidasEnabled, boolean shouldSignWithSHA1) {
-        super(format("healthcheck-response-id-{0}-version-{1}-eidasenabled-{2}-shouldsignwithsha1-{3}", UUID.randomUUID(), msaVersion, eidasEnabled, shouldSignWithSHA1), healthCheckRequestId, entityId, DateTime.now());
+    public HealthCheckResponseFromMatchingService(String entityId, String healthCheckRequestId, String msaVersion, boolean eidasEnabled, boolean shouldSignWithSHA1, boolean universalMatchingDataset) {
```

Comment:
The `eidasEnabled` param can be removed from the constructor as there is no more eIDAS config, and the response string doesn't need to contain the `eidasenabled` bit. Though that may cause issues on the receiving end?